packages/backend/src/errors/step.ts

```diff
@@ -0,0 +1,9 @@
+import { IStepError } from '@plumber/types'
+
+export default class StepError extends Error {
+  constructor(error: IStepError, options?: unknown) {
```

Comment:
Don't use `unknown` unless you have a really good reason

```suggestion
  constructor(error: IStepError, options?: ErrorOptions) {
```